src/constants.js

```diff
@@ -103,4 +103,6 @@ consts.USER_AGENT_LIST = [
     'Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:61.0) Gecko/20100101 Firefox/61.0',
 ];
 
+consts.LOCAL_USER_ID = 'local_user_id';
```

Comment:
Comments please

Reply:
This will probably change.

@mtrunkat said that the user ID could be pulled off the CLI using an env var and replaced with a simple "unknown" or null, if unavailable. @drobnikj Might wanna comment.

Reply:
Yeah, but please can write a comment to `LOCAL_USER_ID` constants to say what it is and where it should be used? Otherwise next person looking at the code will have no idea.